src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/mapping/ClaimMapper.java

```diff
@@ -73,7 +74,8 @@ public Claim map(int index, ResultSet result, StatementContext ctx) throws SQLEx
             null,
             null,
             null,
-            null
+            null,
+            UUID.randomUUID()
```

Comment:
Should be extracted from the result?

Reply:
i changed it to null, from the comment above my understanding is its only used for claim store database which we don't have. Is that right?